public/app.tsx

```diff
@@ -61,12 +75,22 @@ export const FlowFrameworkDashboardsApp = (props: Props) => {
             path={APP_PATH.WORKFLOW_DETAIL}
             render={(
               routeProps: RouteComponentProps<WorkflowDetailRouterProps>
-            ) => <WorkflowDetail {...routeProps} />}
+            ) => (
+              <WorkflowDetail
+                setActionMenu={setHeaderActionMenu}
+                landingDataSourceId={dataSourceId}
```

Comment:
you don't need this props, and I don't see any usage of this props later. I have this for AD because AD has multiple pages. So I need this prop for AD to track the dataSourceId for landing pages to differentiate:
- if the page is opened as a landing page, we select the default remote data source
- if the page is opened by being redirected from another page, we keep the dataSourceId consistent as the previous page instead of selecting the default remote data source. 

None of this is applicable to Flow Framework, so please remove unnecessary and unused props.